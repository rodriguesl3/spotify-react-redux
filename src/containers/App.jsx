import React, { Component } from 'react';

import './App.css';
import thunk from 'redux-thunk';
import { createStore, applyMiddleware, compose } from 'redux';
import { BrowserRouter as Router, Route, Link } from "react-router-dom";
import { Provider } from 'react-redux'
import rootReducers from '../redux-flow/reducers/index'

import { NavBar } from '../components/navbar'
import Login from './Authenticate'

const store = createStore(rootReducers,
  compose(applyMiddleware(thunk), window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__())
);


const App = () => {
  return (
    <Provider store={store}>
      <NavBar brandName="N2L" />
      <div className="container">
        <Router>
          <Route path="/login" component={Login} />
          {/* <Route path="/login" component={Login} />
        <Route path="/main" component={Dashboard} /> */}
        </Router>
      </div>

    </Provider>
  );
}
export default App;
