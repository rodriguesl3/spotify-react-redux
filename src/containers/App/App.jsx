/* eslint-disable no-undef */
/* eslint-disable no-underscore-dangle */
import React from 'react';

import './App.css';
import thunk from 'redux-thunk';
import { createStore, applyMiddleware, compose } from 'redux';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import { Provider } from 'react-redux';
import rootReducers from '../../redux-flow/reducers/index';


import Login from '../Authenticate';
import MainContainer from '../MainContainer/MainContainer';


const store = createStore(rootReducers, compose(applyMiddleware(thunk)));

// const store = createStore(rootReducers,
//   compose(applyMiddleware(thunk),
//     window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()));

const App = () => (
  <Provider store={store}>
    <Router>
      <Switch>
        <div>
          <Route exact path="/" component={Login} />
          <Route path="/main" component={MainContainer} />
        </div>
      </Switch>
    </Router>
  </Provider>
);

export default App;
