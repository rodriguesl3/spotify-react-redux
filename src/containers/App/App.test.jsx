/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('App snaphot test', () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
