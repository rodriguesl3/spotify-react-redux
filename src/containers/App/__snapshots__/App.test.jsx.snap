// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snaphot test 1`] = `
<div
  computedMatch={
    Object {
      "isExact": true,
      "params": Object {},
      "path": "/",
      "url": "/",
    }
  }
  location={
    Object {
      "hash": "",
      "pathname": "/",
      "search": "",
      "state": undefined,
    }
  }
>
  <div>
    <div
      className="content"
    >
      <img
        alt=""
        className="imgUp"
        src="https://wallpapercave.com/wp/EYS4oYf.jpg"
      />
    </div>
    <div
      id="container"
    >
      <img
        alt=""
        className="imgDown"
        src="https://wallpapercave.com/wp/wp2130009.jpg"
      />
      <div
        className="pulse"
        onClick={[Function]}
      />
      <div
        className="dot"
      />
    </div>
  </div>
</div>
`;
